commands/scale_cluster.go

```diff
@@ -201,16 +214,10 @@ func scaleClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 		os.Exit(1)
 	}
 
-	var autoScalingEnabled bool
-	{
-		n, err := util.CompareVersions(clusterDetails.ReleaseVersion, "6.3.0")
-		if err != nil {
-			fmt.Println(color.RedString(err.Error()))
-			os.Exit(1)
-		}
-		if n == 0 || n == 1 {
-			autoScalingEnabled = true
-		}
+	autoScalingEnabled, err := isAutoscalingEnabled(clusterDetails.ReleaseVersion)
+	if err != nil {
+		fmt.Println(color.RedString(err.Error()))
+		os.Exit(1)
```

Comment:
We should stop doing this. The program should only be terminated where it got basically initiated so that start and end are the same at best. Down the road errors should be handled so every caller can decide what and how to do things. This has to improve in the future.

Reply:
I agree, this func is currently called by cobra. There should be a `Runner` similar to how Pawel handled it in devctl. Changing this right now is kind of a refactoring step I want to do later.